packages/api-specification/interface.ts

```diff
@@ -139,19 +138,12 @@ declare namespace ssf {
     blur(): Promise<void>;
 
     /**
-     * Closes the window.
+     * Closes the window. Only works on new windows.
```

Comment:
What is a 'new' window? A window created using the ContainerJS API? Is this only in browser?